src/main/kotlin/domain/MineSweeperGame.kt

```diff
@@ -0,0 +1,43 @@
+package domain
+
+object MineSweeperGame {
+    fun initBoard(weight: Int, height: Int, mineCount: Int): Map<Position, Block> {
```

Comment:
이왕 만든 `MineBoard`를 사용하면 어떨까요?

```suggestion
    fun initBoard(weight: Int, height: Int, mineCount: Int): MineBoard {
```